worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/WorldEditPlugin.java

```diff
@@ -322,38 +319,6 @@ private void setupTags() {
         }
     }
 
-    // FAWE start
-    private void setupBiomes(boolean expectFail) {
```

Comment:
We definitely don't need this any more? P2 still has biome types available during generation?